src/NodeVisitor/UseNamespaceScoperNodeVisitor.php

```diff
@@ -22,6 +22,9 @@ public function __construct($prefix)
     public function enterNode(Node $node)
     {
         if ($node instanceof UseUse) {
+            if ($node->hasAttribute('phpscoper_ignore')) {
```

Comment:
AFAICS it's not specific to the use statement, consider the following:

```php
<?php

namespace A;

function (\Closure $x) {
}
```

or:

```php
<?php

// global namespace

function (Closure $x) {
}
```